modules/nextflow/src/main/groovy/nextflow/executor/GridTaskHandler.groovy

```diff
@@ -237,12 +260,12 @@ class GridTaskHandler extends TaskHandler implements FusionAwareTask {
         return result
     }
 
-    protected String launchCmd0(ProcessBuilder builder, String pipeScript) {
+    protected String submitCmd0(ProcessBuilder builder, String pipeScript) {
         def result = CmdLineHelper.toLine(builder.command())
         if( pipeScript ) {
-            result = "cat << 'LAUNCH_COMMAND_EOF' | ${result}\n"
+            result = "cat << 'SUBMIT_COMMAND_EOF' | ${result}\n"
             result += pipeScript.trim() + '\n'
-            result += 'LAUNCH_COMMAND_EOF\n'
+            result += 'SUBMIT_COMMAND_EOF\n'
```

Comment:
Is there any reason this was changed from launch to submit?

Reply:
e.g. AWS Batch doesn't have a "submit" command because it submits the job via API request instead